contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -795,6 +830,12 @@ contract OCR2DRRegistry is
     }
   }
 
+  modifier onlyWhitelistedUser() {
```

Comment:
We shouldn't use "whitelist" as a name of anything.

Reply:
changed modifer name to `onlyAuthorizedUsers`